src/Tickets/Commerce/Gateways/Stripe/Webhooks/Payment_Intent_Webhook.php

```diff
@@ -34,6 +34,18 @@ public static function handle( array $event, Status_Interface $new_status, \WP_R
 
 		if ( empty( $order ) ) {
 
+			if ( empty( $payment_intent['metadata'][ Payment_Intent::$tc_metadata_identifier ] ) ) {
```

Comment:
We should check for this at the very beginning of the handler method. If this meta doesn't exist then it doesn't make sense to call `$order = tec_tc_get_order( $payment_intent['metadata']['order_id'] );` and `$order = tribe( Order::class )->get_from_gateway_order_id( $payment_intent_id );`

Also, are we sure we want to send $response back with `200` ? I thought in that case we are going to return `null`.

Reply:
This causes an issue when we release because any PI created before this version will not have the proper metadata and will be rejected and clients will lose order updates for those. We can improve on this logic in a later release, when we can be sure all in-flux PIs already have the identifier key.